app/controllers/propertyLinking/ClaimProperty.scala

```diff
@@ -80,16 +80,25 @@ class ClaimProperty @Inject()(
   }
 
   def declareCapacity(uarn: Long, address: String) = authenticatedAction { implicit request =>
-    Ok(views.html.propertyLinking.declareCapacity(DeclareCapacityVM(declareCapacityForm, address, uarn)))
+    Ok(
+      views.html.propertyLinking.declareCapacity(
+        DeclareCapacityVM(declareCapacityForm, address, uarn),
+        backLink(request.headers.get("referer").getOrElse(config.newDashboardUrl("home")))))
```

Comment:
put all the logic in your backLink(..) helper method so you don't duplicate code

Reply:
cheers Darrell - done